game-app/game-core/src/main/java/games/strategy/triplea/odds/calculator/BattleCalculatorPanel.java

```diff
@@ -1052,11 +1054,13 @@ class BattleCalculatorPanel extends JPanel {
     }
     setupAttackerAndDefender();
 
+    final Instant t = Instant.now();
     calculator =
         new ConcurrentBattleCalculator(
             () ->
                 SwingUtilities.invokeLater(
                     () -> {
+                      log.debug("Battle Calculator ready in " + Duration.between(t, Instant.now()));
```

Comment:
Is this change intended or just a remainder of your local debugging?

Reply:
I opted to keep it, since it's just a log line that won't happen often (only when user brings up the battle calc), and let's use measure improvements in a consistent way.